internal/search/types.go

```diff
@@ -251,42 +251,34 @@ type RepoOptions struct {
 
 func (op *RepoOptions) String() string {
 	var b strings.Builder
-	if len(op.RepoFilters) == 0 {
-		b.WriteString("r=[]")
-	}
-	for i, r := range op.RepoFilters {
-		if i != 0 {
-			b.WriteByte(' ')
+	b.WriteString("RepoFilters: ")
+	if len(op.RepoFilters) > 0 {
+		for i, r := range op.RepoFilters {
+			if i != 0 {
+				b.WriteByte(' ')
+			}
+			b.WriteString(strconv.Quote(r))
 		}
-		b.WriteString(strconv.Quote(r))
+	} else {
+		b.WriteString("[]")
 	}
+	b.WriteString("\n")
```

Comment:
The `%q` directive will [auto-escape funky characters](https://play.golang.com/p/tuBha7TorMG) 🙂 This could also be used for `MinusRepoFilters`, which will have the same types of content as `RepoFilters`
```suggestion
	_, _ = fmt.Fprintf(&b, "RepoFilters: %q\n", op.RepoFilters)
```

Reply:
🙏